api_formatter/templates/rest_framework/raw_data_form.html

```diff
@@ -0,0 +1,12 @@
+{% load rest_framework %}
+{% for field in form %}
+    {% if field.name == '_content' %}
+        {{ field.label_tag }}
+        {{ field }}
+    {% elif field.name == '_content_type' %}
+        <div class="select">
+            {{ field.label_tag|add_class:"select__label" }}
+            {{ field|add_class:"select__control" }}
```

Comment:
It turns out `select__label` and `select__control` are not actually classes with styles defined in our library, so we can remove those (and update the Style Guide).